general/src/protocol/contexts.rs

```diff
@@ -670,6 +683,7 @@ fn test_trace_context_roundtrip() {
             );
             map
         },
+        ..Default::default()
```

Comment:
Please add all attributes to test the round trip. In the round trip tests we never use Default.

Reply:
You're right, fixed